web/client/themes/default/less/drawer-menu.less

```diff
@@ -77,6 +77,17 @@
     }
 }
 
+#mapstore-drawermenu.mapstore-mobile-nav {
+    button {
+        box-shadow: none;
```

Comment:
We should remove the shadow to the identify header. This one:
![image](https://user-images.githubusercontent.com/19175505/155335803-701c45b5-0699-4b4a-ae19-4b3caf7e406e.png)